hooks/environment

```diff
@@ -51,7 +50,7 @@ case "${BUILDKITE_AGENT_META_DATA_CONFIG}" in
         # export OMPI_MCA_btl="self,tcp"
         # this is set to avoid rare race conditions on the same node with concurrent (step) jobs
         export OMPI_MCA_orte_tmpdir_base="/tmp/slurm-buildkite/${BUILDKITE_STEP_KEY}"
-        export CLIMATEMACHINE_SETTINGS_OUTPUT_DIR="${CI_BUILD_DIR}/output/${BUILDKITE_STEP_KEY}"
```

Comment:
I think this is used specifically by the ClimateMachine.jl arg parsing

Reply:
Ok, I'll revert this